sdk/v2/core/events.go

```diff
@@ -418,6 +421,22 @@ func (e *eventsClient) DeleteMany(
 	)
 }
 
+func (e *eventsClient) Retry(
+	ctx context.Context,
+	id string,
+) (Event, error) {
+	event := Event{}
+	return event, e.ExecuteRequest(
+		ctx,
+		rm.OutboundRequest{
+			Method:      http.MethodPost,
+			Path:        fmt.Sprintf("v2/events/%s/retry", id),
```

Comment:
REST nit. Since there could tentatively be more than one retry, perhaps this should end with `/retries`?